tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -118,10 +118,14 @@ namespace tensorflow {
 //      take (A, B, A_m, B_m) as input. Among N inputs one can get N!
```

Comment:
Andy suggested earlier (for mkl_optimizer_merge review) to keep all the declarations that are not exported in C++, and only maintain a minimal interface in the header file. The header file is used in graph layout unit test so I have kept only the function required for unit test in the header file.